src/apps/material/material.tsx

```diff
@@ -151,6 +152,10 @@ const Material: React.FC<MaterialProps> = ({ wid }) => {
   const parallelManifestations = materialIsFiction(work) ? manifestations : [];
   const infomediaId = getInfomediaId(currentManifestation);
 
+  // Get disclosure URL parameter from the current URL to see if it should be open
+  const shouldOpenReviewDisclosure =
```

Comment:
we have created a function for this (getUrlQueryParam) that we use in other places. I think you should use that.

Reply:
Perfect addition! Thanks for pointing me the right direction :)